From acfefa36515bf3719cb79b2e0d6118bec25898d1 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Tue, 9 Jul 2024 17:46:46 +0100
Subject: [PATCH] Fix compiler error with clang 17

Redo 7f29110cab6de5598789a98c3a844aca623e3a11, this time without
utterly breaking the indentation.
---
 json-glib/json-scanner.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/json-glib/json-scanner.c b/json-glib/json-scanner.c
index 4ada646..0f940f1 100644
--- a/json-glib/json-scanner.c
+++ b/json-glib/json-scanner.c
@@ -889,6 +889,7 @@ json_scanner_get_token_ll (JsonScanner    *scanner,
 			  break;
 
                         case 'u':
+                        {
                           guint fchar = json_scanner_peek_next_char (scanner);
                           if (is_hex_digit (fchar))
                             {
@@ -994,7 +995,8 @@ json_scanner_get_token_ll (JsonScanner    *scanner,
                               gstring = NULL;
                             }
                           break;
-			  
+                        }
+
 			default:
                           token = JSON_TOKEN_ERROR;
                           value.v_error = JSON_ERROR_TYPE_UNESCAPED_CTRL;
-- 
2.44.0.windows.1

